### ISSUE TITLE
Incompatibility issues with Ruby 3

### ISSUE THREAD

**Issue by ryoung** [DocRaptor/docraptor-ruby#22]
We're using ruby 3.0.1, and this gem fails to work because of the following errors:

```
docraptor-1.4.0/lib/docraptor/configuration.rb:178:in `base_url': undefined method `encode' for URI:Module (NoMethodError)
```

```
docraptor-1.4.0/lib/docraptor/api_client.rb:269:in `build_request_url': undefined method `encode' for URI:Module (NoMethodError)
```

`URI.encode` has been removed in ruby 3

**Comment by benhart** [comment]
I've just stumbled into this now too. I'm not sure who maintains this gem, @JamesPaden @davidronk you've committed over the last year, can we update and release another version?

**Comment by jason-o-matic** [comment]
Hi folks, we're taking a look at this and will comment with updates. Thanks for posting!

**Comment by jason-o-matic** [comment]
This should be fixed if you update to version 2.0.0. Let us know if there are any issues!
